client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -15,12 +15,12 @@
             <div class="row d-md-flex" data-ng-if="!hosting.isExpired">
                 <div class="col-sm-6">
                     <div class="oui-tile mb-5 h-100">
-                        <h3 class="oui-tile__title" data-i18n-static="hosting_dashboard_general_informations"></h3>
+                        <h4 class="oui-header_4 oui-tile__title" data-i18n-static="hosting_dashboard_general_informations"></h4>
```

Comment:
Are you sure you have a parent `h3` tag ?

Reply:
Fixed by using h3 and class oui-header_4 😄